.github/workflows/ci_linux.yml

```diff
@@ -48,6 +48,12 @@ jobs:
               PROJVERSION: "8.2.1",
               allow_failure: "false",
             }
+          - {
+              python: "3.11-dev",
+              GDALVERSION: "3.4.2",
+              PROJVERSION: "8.2.1",
```

Comment:
We could try `GDALVERSION: "3.5.2"` and `PROJVERSION: "9.1.0"` here, but it could break things.

Reply:
Already tried, and it indeed breaks things.

My idea was to first get Python 3.11 working, and then look at updating GDAL in another PR (like #1122).